index.html

```diff
@@ -1678,6 +1678,45 @@ <h3>Bitstring Encoding</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Validity Periods</h3>
+
+      <p>
+The <a data-cite="?VC-DATA-MODEL-2.0#validity-period">validity period</a> of a
+status list is dependent on a variety of factors including:
+      </p>
+      <ul>
+        <li>
+How often do status values change? In real-time? Daily? Weekly? Monthly?
+        </li>
+        <li>
+Is there a regulatory requirement that compels an [=issuer=] to notify a
+[=verifier=] that the status of a [=verifiable credential=] has changed?
+        </li>
+        <li>
+Is there a period of time after which an [=issuer=] would incur reputational
+damage by not notifying a [=verifier=] of a status change?
+        </li>
+        <li>
+Is there an expectation by a [=verifier=] to not accept a
+[=verifiable credential=] with a status list that does not expire for an
+excessive period of time?
+        </li>
+        <li>
+Is there an excessive network bandwidth or computing burden placed upon an
+[=issuer=] or a [=verifier=] if a validity period is too short for a
+status list?
+        </li>
+      </ul>
+
+      <p>
+Since these factors vary based on ecosystem and credential type, there is no
+minimum or maximum validity period that is suggested for a status list.
+[=Issuers=] will need to take various considerations into account that are
+specific to their [=verifiable credential=] types and pick validity periods that
+will strike the right balance in their ecosystem.
```

Comment:
```suggestion
Since these factors vary with the ecosystem and credential type, there is no
minimum or maximum validity period that is suggested for all status lists.
[=Issuers=] will need to consider various factors that are specific to their
[=verifiable credential=] types and choose validity periods that will strike
the right balance in their ecosystem.
```